tests/system_tests_multicast.py

```diff
@@ -673,7 +820,13 @@ def check_if_done(self):
         # sender
         to_rcv = self.n_senders * self.msg_count * self.n_receivers
         if to_rcv == self.n_received and not self.unsettled_deliveries:
-            self.done()
+            if self.done_wait_interval:
+                # start a timer to wait before stopping test in order to
+                # wait for any any duplicate messages to be received
+                self.done_timer = threading.Timer(self.done_wait_interval, self.done)
```

Comment:
You could use a `poll_timer` like here - https://github.com/skupperproject/skupper-router/blob/main/tests/system_tests_delivery_counts.py#L954 - instead of `threading.Timer`

Reply:
Removed done_timer.
Removed `import threading`.
Added poll_timer.